src/Entity/User.php

```diff
@@ -1208,6 +1208,13 @@ public function isInvitedToRelay(Centre $relay): bool
         return null !== $userRelay && !$userRelay->getBValid();
     }
 
+    public function hasValidLinkToRelay(Centre $relay): bool
+    {
+        $userRelay = $this->getSubjectRelaysForRelay($relay);
+
+        return null !== $userRelay && $userRelay->getBValid();
```

Comment:
`getSubjectRelaysForRelay` this name make me think it should return a collection or array, so should'nt we test if its not empty rather than not null ?